mne_bids/mne_bids.py

```diff
@@ -537,8 +618,16 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
     config : str | None
         A path to the configuration file to use if the data is from a BTi
         system.
-    overwrite : bool
-        If the file already exists, whether to overwrite it.
+    write_mode : str, one of ('append', 'overwrite', 'error')
+        How any existing files should be handled when new data is produced.
+        Defaults to 'append'.
+        - 'append': Any file that already exists will be overridden with new
+            data, except `participants.tsv` and `scans.tsv` which will have the
+            new data appended.
+        - 'overwrite': Completely clear the session level BIDS directory,
```

Comment:
what do you mean by "session level BIDS directory"? I remember I had used this term before but now I'm confused :)

Reply:
By session level I mean the folder `ses-<session>`. It is this folder that is re-written by the `make_bids_folders` function if `write_mode = 'overwrite'`.